swupd/packs_test.go

```diff
@@ -537,3 +537,106 @@ func TestTwoDeltasForTheSameTarget(t *testing.T) {
 		ts.checkExists(fmt.Sprintf("www/20/delta/10-20-%s-%s", hashA, hashB))
 	}
 }
+
+func TestPackRenames(t *testing.T) {
+	ts := newTestSwupd(t, "packing-renames-")
+	defer ts.cleanup()
+
+	ts.Bundles = []string{"os-core"}
+
+	content := strings.Repeat("CONTENT", 1000)
```

Comment:
much larger than it needs to be to catch a rename/delta.

Reply:
Gets deleted after test runs though, es ok.